01-node-tutorial/answers/09-path-module.js

```diff
@@ -0,0 +1,12 @@
+const path = require("path");
+
+console.log("Path separator: ", path.sep);
+
+const filePath = path.join("/content/", "subfolder", "test.txt");
+console.log("File path: ", filePath);
+
+const base = path.basename(filePath);
+console.log("Base name: ", base);
+
+const absolute = path.resolve(__dirname, "/content/", "subfolder", "test.txt");
```

Comment:
Using "/content/" produces a different path (not absolute) than using "content"  EX:
const absolute = path.resolve(__dirname, "content", "subfolder", "test.txt");
console.log("Absolute path: ", absolute);

Not a big deal, but try both out and see what the output is.

Reply:
Thank you for your comment! I've checked both and see the difference now